src/riak_ts.proto

```diff
@@ -111,6 +112,7 @@ message TsCell {
   optional sint64 timestamp_value = 3;
   optional bool boolean_value = 4;
   optional double double_value = 5;
+  optional bytes blob_value = 6;
```

Comment:
I did talk to @lukebakken a while back about putting the blob data into the varchar column to avoid an extra record field per row, did that have problems?